tests/all/js_globals/Symbol.rs

```diff
@@ -0,0 +1,41 @@
+#![allow(non_snake_case)]
+
+use project;
+
+#[test]
+fn has_instance() {
+    project()
+        .file(
+            "src/lib.rs",
+            r#"
+            #![feature(proc_macro, wasm_custom_section)]
+
+            extern crate wasm_bindgen;
+            use wasm_bindgen::prelude::*;
+            use wasm_bindgen::js;
+
+            #[wasm_bindgen]
+            pub fn symbol_has_instance() -> js::Symbol {
+                js::Symbol::has_instance()
+            }
+
+        "#,
+        )
+        .file(
+            "test.ts",
```

Comment:
TypeScript has been temporarily disabled, so this test fails with `thread 'js_globals::Symbol::has_instance' panicked at 'webpack needs to be enabled to use typescript'`. Could you please change that line to `test.js`?

Reply:
Sorry,  I didn't realize it 😅